draft-ietf-ppm-dap-taskprov.md

```diff
@@ -240,7 +240,7 @@ opaque `dp_config` contains the following structure:
 enum {
     reserved(0), /* Reserved for testing purposes */
     none(1),
-    (255)
+    (2^32-1)
```

Comment:
Can you also make the corresponding change in IANA Considerations? "one-byte identifier" -> "four-byte identifier"